standard/variables.md

```diff
@@ -72,7 +70,7 @@ For the purpose of definite assignment checking, an array element is considered
 
 ### 9.2.5 Value parameters
 
-A parameter declared without a `ref` or `out` modifier is a ***value parameter***.
+A parameter declared without an `in`, `out`, or `ref` modifier is a ***value parameter***.
```

Comment:
After this clause would be where to introduce the concept of pass-by-reference to encompass *reference parameters*, *output parameters* and *input parameters*.